flutter_local_notifications_windows/lib/src/plugin/stub.dart

```diff
@@ -96,4 +96,7 @@ class FlutterLocalNotificationsWindows extends WindowsNotificationsBase {
   @override
   @visibleForTesting
   void enableMultithreading() {}
```

Comment:
Based on your recent findings, is this method still needed?

Reply:
Not _needed_, as we can use `-j 1`, but I don't think that solves the fundamental problem of concurrency at hand. I can remove it though as it really shouldn't be needed -- or feel free to remove it yourself if you'd like.

Reply:
Removed